shared/util/quit-helper.desktop.js

```diff
@@ -27,17 +30,23 @@ function isMainThread () {
 }
 
 function _executeActions (actions: Array<Action>) {
+  // Don't allow us to re-enter this logic if we're already shutting down
+  if (isQuitting) {
```

Comment:
If it didn't work the first time why would it the second?

On Friday, July 15, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/util/quit-helper.desktop.js
> <https://github.com/keybase/client/pull/3469#discussion_r71046465>:
>
> > @@ -27,17 +30,23 @@ function isMainThread () {
> >  }
> >
> >  function _executeActions (actions: Array<Action>) {
> > +  // Don't allow us to re-enter this logic if we're already shutting down
> > +  if (isQuitting) {
>
> If something fails while quitting and this becomes true, can the user
> never quit anymore?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3469/files/696a85227ca0f7720ffeeea0c9eb4a149973b035#r71046465>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhU5_Mn9VlIrPJ-QnTF7hZTENmrAQnks5qWAH9gaJpZM4JMsjQ>
> .
>